client/js/screens/Login/styles.js

```diff
@@ -1,8 +1,11 @@
 import { StyleSheet } from 'react-native';
-import { container } from '../../config/styles';
+import { container, colors, padding } from '../../config/styles';
 
 const styles = StyleSheet.create({
-  container: { ...container }
+  container: { ...container },
+  button: {
+    width: 125
```

Comment:
use flexbox instead of giving fix widths and heights to show your component the same size on all phones.

Reply:
Removed width and replaced it with flexbox.